src/profile-logic/profile-tree.js

```diff
@@ -158,25 +158,50 @@ export type ProfileTreeClass = ProfileTree;
 
 export function getCallTree(
   thread: Thread, interval: Milliseconds, funcStackInfo: FuncStackInfo,
-  implementationFilter: string
+  implementationFilter: string, invertCallstack: boolean
 ): ProfileTree {
   return timeCode('getCallTree', () => {
     const { funcStackTable, stackIndexToFuncStackIndex } = funcStackInfo;
     const sampleFuncStacks = getSampleFuncStacks(thread.samples, stackIndexToFuncStackIndex);
 
     const funcStackSelfTime = new Float32Array(funcStackTable.length);
     const funcStackTotalTime = new Float32Array(funcStackTable.length);
+    let funcStackToRoot;
```

Comment:
Could you move the variable declarations to the if block and use const? I'm not seeing why they need to be in the outer lexical scope.